endless/src/endless/EndlessUsbToolDlg.cpp

```diff
@@ -5131,6 +5134,31 @@ bool CEndlessUsbToolDlg::CopyFilesToESP(const CString &fromFolder, const CString
 	return retResult;
 }
 
+bool CEndlessUsbToolDlg::CreatePersistentStorageFileOnexFAT(const CString& drive)
+{
+    HANDLE tmpfile;
+    CString storage = drive + L"\\endless\\persistent.img";
+    char marker[] = "endless_live_storage_marker";
+    uint64_t bytes;
+    ULARGE_INTEGER free_bytes;
+    DWORD written;
+
+    if (!GetDiskFreeSpaceEx(drive, &free_bytes, NULL, NULL))
+        return FALSE;
```

Comment:
A fun fact about C++ and MFC is that strictly speaking `FALSE` and `false` mean different things, and `false` should be used when the function returns `bool` (as this one does). However I'm sure it has literally zero effect either way. Surely the compiler is smart enough not to do an explicit conversion from `(int) 0` to `false`.

Reply:
I hate computers.

Reply:
Should we open a ticket to fix the other returns of FALSE from bool functions in EndlessUsbToolDlg.cpp as well?

Reply:
I don't think this strictly needs changing, particularly if the compiler isn't complaining. I just thought you'd appreciate it...